persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/StructuralVariantMapper.xml

```diff
@@ -159,6 +159,18 @@
         </if>
     </sql>
 
+    <sql id="whereWithStructalVariants">
```

Comment:
We moeten er hier voor zorgen dat we ook:
1. Alle genes op een positie kunnen zoeken.
2. Null gene op een positie kunnen zoeken.

Reply:
Should now be possible in [4c93d75](https://github.com/cBioPortal/cbioportal/commit/4c93d75879595db2e4add0ff8452a5647b540756)